src/components/Input/Dropdown.tsx

```diff
@@ -19,28 +20,27 @@ import { animationTransition } from "../../utils/animations/config";
  */
 const DropdownOptions = ({
   options,
-  noShowSelected,
   selectedItemValue,
   optionOnClick,
   noOptionsText,
 }: {
   options: Array<DropdownOptionType>;
-  noShowSelected: boolean;
   selectedItemValue: any;
-  optionOnClick: (optionValue: any) => void;
+  optionOnClick: (optionValue: any) => any;
```

Comment:
I am not sure about react's standard but I don't think the prop function can return anything

Reply:
I don’t know what you mean by “React’s standard”, but I might just keep it `void` anyway because that worked fine.